vhost-device-sound/src/audio_backends/pipewire.rs

```diff
@@ -84,7 +96,7 @@ pub struct PwBackend {
 }
 
 impl PwBackend {
-    pub fn new(stream_params: Arc<RwLock<Vec<Stream>>>) -> Self {
+    pub fn new(stream_params: Arc<RwLock<Vec<Stream>>>) -> std::result::Result<Self, PwError> {
```

Comment:
instead of `std::result::Result<Self, PwError>` why not import it and use the type alias here?

Reply:
`Result` is imported from `crate` root above in the imports, there's no alias for `std::result::Result<Self, PwError>`

Reply:
right, I meant to create a type alias for it, something like `type PwBackendResult = std::result::Result<PwBackend, PwError>;`

Reply:
No need to for just one instance of it, I think